### PR TITLE
sexplib0 v0.15.1

### PR DESCRIPTION
**Pull request by cwong-ocaml**
Signed-off-by: Cameron Wong <cwong@janestreet.com>

### CODE REVIEW

File: packages/sexplib0/sexplib0.v0.15.1/opam [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+homepage: "https://github.com/janestreet/sexplib0"
+bug-reports: "https://github.com/janestreet/sexplib0/issues"
+dev-repo: "git+https://github.com/janestreet/sexplib0.git"
+doc: "https://ocaml.janestreet.com/ocaml-core/latest/doc/sexplib0/index.html"
+license: "MIT"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.04.2"}
```
Comment (kit-ty-kate) [inline packages/sexplib0/sexplib0.v0.15.1/opam:13]:
```suggestion
  "ocaml" {>= "4.08"}
```
```
#=== ERROR while compiling sexplib0.v0.15.1 ===================================#
# context              2.1.2 | linux/x86_64 | ocaml-base-compiler.4.07.1 | pinned(https://github.com/janestreet/sexplib0/archive/refs/tags/v0.15.1.tar.gz)
# path                 ~/.opam/4.07/.opam-switch/build/sexplib0.v0.15.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p sexplib0 -j 31
# exit-code            1
# env-file             ~/.opam/log/sexplib0-2478-eef24b.env
# output-file          ~/.opam/log/sexplib0-2478-eef24b.out
### output ###
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.sexplib0.objs/byte -intf-suffix .ml -no-alias-deps -open Sexplib0__ -o src/.sexplib0.objs/byte/sexplib0__Sexp_conv.cmo -c -impl src/sexp_conv.ml)
# File "src/sexp_conv.ml", line 96, characters 17-45:
# Error: Unbound module Obj.Extension_constructor
# (cd _build/default && /home/opam/.opam/4.07/bin/ocamlopt.opt -w -40 -g -I src/.sexplib0.objs/byte -I src/.sexplib0.objs/native -intf-suffix .ml -no-alias-deps -open Sexplib0__ -o src/.sexplib0.objs/native/sexplib0__Sexp_conv.cmx -c -impl src/sexp_conv.ml)
# File "src/sexp_conv.ml", line 96, characters 17-45:
# Error: Unbound module Obj.Extension_constructor
```

**Comment by kit-ty-kate** [comment]
Thanks a lot! Could you return the fix to the opam file upstream?

**Comment by cwong-ocaml** [comment]
I can do that, but it's not clear to me how to do this without messing up the point release we just merged.

Should I just change the opam file in the branch without making a corresponding tag?

**Comment by mseri** [comment]
That's right, there is no need to re-tag or re-release. Opam will use the file committed to this repository in any case

**Comment by cwong-ocaml** [comment]
done!

**Comment by kit-ty-kate** [comment]
Thanks! It might be useful to also port it to the `master` branch so it does not cause issues in the future for v0.16

**Pull request merged**